test/lan_udev_auto_test.rb

```diff
@@ -4,9 +4,12 @@
 
 require "yast"
 
-module Yast
+include Yast
```

Comment:
ah, please do not do this. It populates whole global namespace and can cause troubles to other tests.